web.py

```diff
@@ -168,16 +168,34 @@ def get_pokemarkers():
     session.close()
 
     for pokemon in pokemons:
-        markers.append({
-            'id': 'pokemon-{}'.format(pokemon.id),
-            'type': 'pokemon',
-            'trash': pokemon.pokemon_id in config.TRASH_IDS,
-            'name': POKEMON_NAMES[pokemon.pokemon_id],
-            'pokemon_id': pokemon.pokemon_id,
-            'lat': pokemon.lat,
-            'lon': pokemon.lon,
-            'expires_at': pokemon.expire_timestamp,
-        })
+            content = {
+                'id': 'pokemon-{}'.format(pokemon.id),
+                'type': 'pokemon',
+                'trash': pokemon.pokemon_id in config.TRASH_IDS,
+                'name': POKEMON_NAMES[pokemon.pokemon_id],
+                'pokemon_id': pokemon.pokemon_id,
+                'lat': pokemon.lat,
+                'lon': pokemon.lon,
+                'expires_at': pokemon.expire_timestamp,
+            }
+            
+            try:
+                if pokemon.atk_iv is not None:
+                    iv = {
+                        'atk': pokemon.atk_iv,
+                        'def': pokemon.def_iv,
+                        'sta': pokemon.sta_iv,
+                        'move1': MOVES.get(pokemon.move_1, {}).get('name'),
+                        'move2': MOVES.get(pokemon.move_2, {}).get('name'),
+                    }
+                    content.update(iv)
+                
+                
+            except KeyError:
```

Comment:
Good to know, I'll change it.

but we need to call any of the ivs then to get an error then. Not just the array assignment and update command, since that passes but still just assigns them as null

code without the `if pokemon.atk_iv is not none:` produced /data looking like this.

```
  {
    "atk": null, 
    "def": null, 
    "expires_at": 1485177909, 
    "id": "pokemon-735140", 
    "lat": 64.1487, 
    "lon": -21.7964, 
    "move1": null, 
    "move2": null, 
    "name": "Zubat", 
    "normal": false, 
    "pokemon_id": 41, 
    "sta": null, 
    "trash": true, 
    "type": "pokemon"
  }, 
```